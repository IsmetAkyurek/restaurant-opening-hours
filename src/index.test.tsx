import * as index from ".";

describe("Index", () => {
  it("Should render app withour crashing", () => {
    expect(JSON.stringify(Object.assign({}, index, { _reactInternalInstance: "censored" }))).toMatchSnapshot();
  });
});
