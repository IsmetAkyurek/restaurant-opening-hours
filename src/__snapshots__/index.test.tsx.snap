// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Should render app withour crashing 1`] = `"{\\"default\\":{},\\"_reactInternalInstance\\":\\"censored\\"}"`;
